content/blog/2016/2016-09-08-continuous-delivery-of-infra.adoc

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Continuous Delivery of Infrastructure with Jenkins"
+tags:
+- event
+- JenkinsWorld
+author: rtyler
+---
+
+NOTE: This is a guest post by Jenkins World speaker
+link:https://github.com/rtyler[R Tyler Croy], infrastructure maintainer for the
+Jenkins project.
+
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World, role=right]
+
+I don't think I have ever met a tools, infrastructure, or operations team that
+did not have a ton of work to do. The Jenkins project's
+link:https://github.com/jenkins-infra[infrastructure]
+"team" is no different; too much work, not enough time. In lieu of hiring more
+people, which isn't always an option, I have found heavy automation and
+continuous delivery pipelines to be two solutions within reach of the
+over-worked infrastructure team.
+
+
+As a big believer in the concept of "Infrastructure as Code", I have been,
+slwoly but surely, introducing more and more code to define the Jenkins
+project's infrastructure. Whether implemented in our
+link:https://github.com/jenkins-infra/jenkins-infra[Puppet code-base],
+link:https://github.com/jenkins-infra/ircbot[Docker containers],
+or even as
+link:https://github.com/jenkins-infra/azure/tree/master/machines[machine specifications]
+with
+link:http://packer.io[Packer].
```

Comment:
"...infrastructure.  Whether ... Packer. The..." 

This is one long lead-in, so it would be:
1 - "...infrastructure.  Whether ... Packer, the..."
or possibly:
2 - "...infrastructure,  whether ... Packer.  The..." 

But I'd suggest 1.